rust/src/sip/detect.rs

```diff
@@ -367,14 +357,16 @@ unsafe extern "C" fn sip_from_hdr_get(
 unsafe extern "C" fn sip_from_hdr_get_data(
     tx: *const c_void, flow_flags: u8, local_id: u32, buffer: *mut *const u8, buffer_len: *mut u32,
 ) -> bool {
-    sip_get_header_value(
-        tx,
-        local_id,
-        flow_flags,
-        buffer,
-        buffer_len,
-        "From\0".as_ptr() as *const c_char,
-    )
+    let tx = cast_pointer!(tx, SIPTransaction);
+    if let Some(value) = sip_get_header_value(tx, local_id, flow_flags.into(), "From") {
+        *buffer = value.as_ptr();
+        *buffer_len = value.len() as u32;
+        return true;
+    }
+    // else
```

Comment:
please get rid of these comments (also below)

Reply:
ok